.github/workflows/repotests.yml

```diff
@@ -136,6 +137,19 @@ jobs:
           # mv *.hpi jenkins
           # CDXGEN_DEBUG_MODE=debug bin/cdxgen.js -p -r -t jenkins jenkins -o bomresults/bom-jenkins.json --validate
           ls -ltr bomresults
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
@prabhu Might be simpler and easier to maintain to just specify shell: bash instead of having separate command sections for linux and windows.

Reply:
Good point